task2-3/createAndFillDB.sql

```diff
@@ -8,6 +10,20 @@ CREATE TABLE users (
 	is_deleted boolean
 );
 
+CREATE TABLE groups (
+	id text PRIMARY KEY,
+	name text,
+	permissions text []
```

Comment:
let's discuss this line `permissions text []`. what is this operator `[]` used for?
if you are trying to save an array in the table then this is not correct. You should create another table for this

Reply:
Here I have a question, why is it not correct using ```[]``` ? As I see it is works just fine in MySQL

Reply:
![image](https://user-images.githubusercontent.com/22420612/217904874-1571ace9-c0f2-45d8-8ef7-0c7112c6ba29.png)

Reply:
Or it is better to do another table with permissions and then add one more bridge table like **groups_permissions**